apps/portals/index.html

```diff
@@ -2,40 +2,31 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.svg" />
+    <link rel="shortcut icon" href="/favicon.svg" />
 
     <!-- Facebook Meta Tags -->
-    <meta property="og:url" content="%PRODUCTION_URL%" />
+    <meta property="og:url" content="https://%VITE_PORTAL_KEY%.synapse.org/" />
     <meta property="og:type" content="website">
-    <meta property="og:title" content="%PORTAL_NAME%" />
-    <meta property="og:description" content="%PORTAL_DESCRIPTION%" />
-    <meta name="image" property="og:image" content="%PRODUCTION_URL%/socialmedia.png" />
+    <meta property="og:title" content="%VITE_PORTAL_NAME%" />
+    <meta property="og:description" content="%VITE_PORTAL_DESCRIPTION%" />
+    <meta name="image" property="og:image" content="https://%VITE_PORTAL_KEY%.synapse.org/socialmedia.png" />
```

Comment:
IIRC, during testing I found that these references must be absolute (rather than relative paths), so I think your approach should work.